app/lib/constants.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
     PERIOD_END: 'PeriodEnd',
     SUPPLIER: 'Supplier',
   },
-  OUTPUT_DIR: 'site/json',
+  OUTPUT_DIR: 'output',
+  CONTAINER_NAME: process.env.CONTAINER_NAME || 'etl-output'
```

Comment:
I think its worth having an account per process so there is no danger of overwriting between processes. The change for that is just config but it'll require the `README` to be updated so i'll wait for that till I merge this.